### DISCUSSION TITLE
An invitation to a crucial Team Leads meeting for this Friday

### DISCUSSION THREAD

**Discussion by Rayccheal** [clentafrica/hr-people-operations#9] [Announcements]
Dear Team Leads, @clentafrica/wg-marketing-comms , @clentafrica/wg-design , @clentafrica/wg-research-and-data , @clentafrica/wg-programs , @clentafrica/wg-community-managers , @clentafrica/wg-sustainability-esg , @clentafrica/wg-education , @clentafrica/wg-hr , @clentafrica/wg-web-dev , @clentafrica/wg-disability-gender-equality , @clentafrica/wg-gsoc , @clentafrica/wg-energy , @clentafrica/wg-strategy , @clentafrica/wg-partnerships , @clentafrica/wg-fundraising , @clentafrica/wg-fundraising @yigakpoa

I hope this finds you well!

I would like to invite you to a groundbreaking Team Leads meeting this Saturday, 12/10/2024, from 10:00 to 10:30 AM WAT. 

The purpose of this meeting is to discuss important updates and initiatives that will impact our teams. Your insights and contributions will be essential to our discussions.

Please confirm your availability for this meeting, letting us know if you would have any scheduling conflicts.

Thank you, and I look forward to our collaboration.

PS: Please feel free to respond here, even if you've already confirmed via email, as we're working to finalize a time that will accommodate everyone's availability. 

See you in the comments!

**Comment by Fathalrhman26** [comment]
Confirmed!, I will be available.

**Reply by yigakpoa** [reply]
Please vote for the time here: https://github.com/clentafrica/hr-people-operations/discussions/10#discussioncomment-10897264

**Comment by kola42** [comment]
Hello Team,

Mail acknowledged, this is to confirm my availability.

Regards
Kolawole.

**Comment by MadhalasaSJ** [comment]
I will be available, Thank you!!

**Comment by Jonniie** [comment]
I'll be available

**Comment by SDauphin15** [comment]
Good morning,


 I am not available on weekends

**Comment by ALIPHATICHYD** [comment]
Hello Team,

Mail acknowledged, this is to confirm my availability.

Regards
Fuad ALIPHATIC.

**Comment by aramide-dess** [comment]
Hello.
Mail received. Yes I will be available.

**Comment by Ladique** [comment]
I'll be available

**Comment by Onodjayeke** [comment]
Hi Team😊,

I just want to take a moment to say a heartfelt thank you for all your positive responses. It's really encouraging to see how invested you all are in what we’re trying to achieve together. I know we’re all juggling a lot, so I truly appreciate you making the time for this meeting.

This Saturday’s session will be short but impactful, and I’m excited about the ideas and insights we’re going to share. Your contributions mean a lot, and I’m confident that together we’ll be able to move forward with clarity and purpose.

Please don’t hesitate to reach out if anything comes up before the meeting. I completely understand if things change. Otherwise, I’m really looking forward to working with each of you as we navigate these next steps.

Thanks again, and I’m excited to connect with you all on Saturday❤️🥰.

Take care,
Benanemi 
(HR cordinator)

**Comment by EmmanetD** [comment]
Hello Team,

Thank you for the invitation. I will be available for the meeting.

Regards,
Damilare 
(PM Team Lead)

**Reply by Okpeelizabeth1989** [reply]
Yes I will be available for the meeting

**Comment by mike-oye** [comment]
I will be available.

**Comment by Rayccheal** [comment]
Hello Team,

Thank you for your prompt feedback! The team and I have found a new date and time that we believe better accommodates everyone’s schedule, and I will be sharing that soon. To save us time, however, we may consider using a poll this time. We truly appreciate your input!

Thank you for your understanding!

**Reply by yigakpoa** [reply]
Hi Racheal how about creating a poll here for contributors to vote instead of leaving individual messages. Then we can schedule the call with one of our zoom links for the time with the most votes.

Please keep in mind that we have people in the diaspora. so afternoons GMT are usually better for contributors in India, USA, and Africa. 3-6pm GMT+1 is our bets bet.

 What do we thing @clentafrica/wg-hr

**Reply by Rayccheal** [reply]
Hi Yiga! I think it is an excellent idea! I would run with that now, and create a poll immediately, having the suggested time in mind.

Thank you!

**Comment by Maxewa** [comment]
I will be available

**Comment by yigakpoa** [comment]
**Please find the discussion with a poll for this meeting here: https://github.com/clentafrica/hr-people-operations/discussions/10**

**Comment by mbomdaS** [comment]
Ok I will be there.

**Reply by Rayccheal** [reply]
Hellloooo!

Please vote for the time here #10

**Comment by uchiha-vivek** [comment]
Yeah i will be available

**Reply by Rayccheal** [reply]
Please, vote for meeting time here #10

**Comment by emylajem** [comment]
I will be available.

Please I haven't attended any of the meetings before. I need to be updated
about my role. Where do I belong?

**Reply by Rayccheal** [reply]
Hi @emylajem, 

We look forward to discussing updates on your roles in the upcoming meeting. A Zoom link will be shared shortly, once we finalize the time based on votes.

Please take a moment to vote here: #10 

Thanks for staying engaged!

**Comment by JuliaKamila02** [comment]
Hello,

Unfortunately I won’t be available this weekend. When will another meeting
take place?

I am also new here and don’t really know where I belong, can maybe someone
guide me as to what my tasks/role are?

Thank you
Julia M

**Reply by Rayccheal** [reply]
Hi Julia,

Thanks for your feedback!

Attending this meeting should help you find your place here. We’re considering moving it to Friday instead of Saturday. Would that not work for you still?

Once you decide, please vote here: #10, as we look forward to seeing you on Friday!

Regards,
Racheal

**Comment by Abraham100000** [comment]
Hi all,
I should be available for the meeting.

Regards

**Reply by Rayccheal** [reply]
Hi Abraham, 

Please vote here #10

**Comment by Obutorbeatrice** [comment]
I will be available, I haven't attended any of the meetings before. Can I
be briefed aforetime please?

On Thu, 10 Oct 2024 at 09:43, Abraham100000 ***@***.***>
wrote:

> Hi all,
> I should be available for the meeting.
>
> Regards
>
> On Thu, Oct 10, 2024 at 9:39 AM JuliaKamila02 ***@***.***>
> wrote:
>
> > Hello,
> >
> > Unfortunately I won’t be available this weekend. When will another
> meeting
> > take place?
> >
> > I am also new here and don’t really know where I belong, can maybe
> someone
> > guide me as to what my tasks/role are?
> >
> > Thank you
> > Julia M
> >
> > On Wed 9. Oct 2024 at 17:29, Rayccheal ***@***.***> wrote:
> >
> > > Dear Team Leads, @clentafrica/wg-marketing-comms
> > > <https://github.com/orgs/clentafrica/teams/wg-marketing-comms> ,
> > > @clentafrica/wg-design
> > > <https://github.com/orgs/clentafrica/teams/wg-design> ,
> > > @clentafrica/wg-research-and-data
> > > <https://github.com/orgs/clentafrica/teams/wg-research-and-data> ,
> > > @clentafrica/wg-programs
> > > <https://github.com/orgs/clentafrica/teams/wg-programs> ,
> > > @clentafrica/wg-community-managers
> > > <https://github.com/orgs/clentafrica/teams/wg-community-managers> ,
> > > @clentafrica/wg-sustainability-esg
> > > <https://github.com/orgs/clentafrica/teams/wg-sustainability-esg> ,
> > > @clentafrica/wg-education
> > > <https://github.com/orgs/clentafrica/teams/wg-education> ,
> > > @clentafrica/wg-hr <https://github.com/orgs/clentafrica/teams/wg-hr>
> ,
> > > @clentafrica/wg-web-dev
> > > <https://github.com/orgs/clentafrica/teams/wg-web-dev> ,
> > > @clentafrica/wg-disability-gender-equality
> > > <
> https://github.com/orgs/clentafrica/teams/wg-disability-gender-equality>
> >
> > > , @clentafrica/wg-gsoc <
> > https://github.com/orgs/clentafrica/teams/wg-gsoc>
> > > , @clentafrica/wg-energy
> > > <https://github.com/orgs/clentafrica/teams/wg-energy> ,
> > > @clentafrica/wg-strategy
> > > <https://github.com/orgs/clentafrica/teams/wg-strategy> ,
> > > @clentafrica/wg-partnerships
> > > <https://github.com/orgs/clentafrica/teams/wg-partnerships> ,
> > > @clentafrica/wg-fundraising
> > > <https://github.com/orgs/clentafrica/teams/wg-fundraising> ,
> > > @clentafrica/wg-fundraising
> > > <https://github.com/orgs/clentafrica/teams/wg-fundraising> @yigakpoa
> > > <https://github.com/yigakpoa>
> > >
> > > I hope this finds you well!
> > >
> > > I would like to invite you to a groundbreaking Team Leads meeting this
> > > Saturday, 12/10/2024, from 10:00 to 10:30 AM WAT.
> > >
> > > The purpose of this meeting is to discuss important updates and
> > > initiatives that will impact our teams. Your insights and
> contributions
> > > will be essential to our discussions.
> > >
> > > Please confirm your availability for this meeting, letting us know if
> > you
> > > would have any scheduling conflicts.
> > >
> > > Thank you, and I look forward to our collaboration.
> > >
> > > PS: Please feel free to respond here, even if you've already confirmed
> > via
> > > email, as we're working to finalize a time that will accommodate
> > everyone's
> > > availability.
> > >
> > > See you in the comments!
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <https://github.com/clentafrica/hr-people-operations/discussions/9>,
> or
> > > unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/BLOEL5UTQNRM4Y43DLMFWXDZ2VDWBAVCNFSM6AAAAABPUZ5ZFCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGI4TKOJYG4>
>
> >
> > > .
> > > You are receiving this because you are on a team that was
> > mentioned.Message
> > > ID: ***@***.***>
> > >
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/clentafrica/hr-people-operations/discussions/9#discussioncomment-10901454>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/BKCAJKO5TTFEM4LDK5XNHVLZ2Y4MTAVCNFSM6AAAAABPUZ5ZFCVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAOJQGE2DKNA>
>
> > .
> > You are receiving this because you are on a team that was
> mentioned.Message
> > ID:
> <clentafrica/hr-people-operations/repo-discussions/9/comments/10901454
> > @github.com>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/clentafrica/hr-people-operations/discussions/9#discussioncomment-10901500>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BJ3CXF7CKRGL772EM4ITS7DZ2Y44HAVCNFSM6AAAAABPUZ5ZFCVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAOJQGE2TAMA>
> .
> You are receiving this because you are on a team that was mentioned.Message
> ID: <clentafrica/hr-people-operations/repo-discussions/9/comments/10901500
> @github.com>
>


-- 
THE SINGING *BETARICE*!

**Reply by Rayccheal** [reply]
Thank you for your feedback, Beatrice!

A Zoom link will be sent as soon as we all decide on a meeting time here #10 

Please vote for time here #10. We look forward to having you join us!

PS: No prior briefing is needed—just join us on Friday!

Regards,
Racheal

**Comment by AxelleDongni** [comment]
Hi everyone,
I will be available.

**Reply by Rayccheal** [reply]
Please vote here #10
